config/samples/dataplane_v1beta1_openstackdataplanenodeset_ceph.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: dataplane.openstack.org/v1beta1
 kind: OpenStackDataPlaneNodeSet
 metadata:
-  name: openstack-edpm-ceph
```

Comment:
why?

Reply:
It would be nice to keep the name. The other samples have the name.

MTU comes up for Ceph in the HCI scenario since it benefits from jumbo frames.

Reply:
Not sure, I think I copied the sample during the crd-proposal change where I had renamed the ipam files to better distinguish between the file contents.

I reverted this anyway.